ProcessMaker/Repositories/CaseParticipatedRepository.php

```diff
@@ -65,14 +84,33 @@ public function update(CaseStarted $case, TokenInterface $token)
                 return;
             }
 
+            $processData = [
```

Comment:
this section is repeated below, maybe it's better abstract in a method

Reply:
It was fixed